changedetectionio/processors/text_json_diff.py

```diff
@@ -118,7 +118,7 @@ def run_changedetection(self, uuid, skip_when_checksum_same=True):
         include_filters_from_tags = self.datastore.get_tag_overrides_for_watch(uuid=uuid, attr='include_filters')
 
         # 1845 - remove duplicated filters in both group and watch include filter
-        include_filters_rule = list({*watch.get('include_filters', []), *include_filters_from_tags})
+        include_filters_rule = list(dict.fromkeys(watch.get('include_filters', []) + include_filters_from_tags))
```

Comment:
`list(dict.fromkeys([*watch.get('include_filters', []), *include_filters_from_tags]))` is also possible.

Reply:
```
import timeit

def timeitext(num_times):
    case1 = timeit.timeit(
        '[["safd", "axcv"], *["tbnsdfb", "afvavdas"]]',
        number=num_times,
    )
    case2 = timeit.timeit(
        '["safd", "axcv"] + ["tbnsdfb", "afvavdas"]',
        number=num_times,
    )
    return f"""{fstring=}
{formattim=}
"""

print(timeitext(1000000))
```

results 

```
case1=0.13575055799992697
case2=0.10457201000099303
```

So if it is always a list type, it(`+`) will be slightly better.